requirements.txt

```diff
@@ -0,0 +1,7 @@
+Flask
+Flask-SQLAlchemy
+Flask-Cors
+PyCDS==2.0.1
+# Testing requirements
```

Comment:
Some of our repos include testing requirements in `requirements.txt`, others separate them into `test_requirements.txt`. This repo has both. We should decide which style we prefer.